pkg/volume/plugins.go

```diff
@@ -55,6 +55,10 @@ type VolumeOptions struct {
 	ClusterName string
 	// Tags to attach to the real volume in the cloud provider - e.g. AWS EBS
 	CloudTags *map[string]string
+	// ProvisioningOptions are additional options for the provisioner, e.g.
+	// to create a volume of certain QoS characteristics. These options are
+	// opaque to anything but appropriate provisioner plugin.
+	ProvisioningOptions string
```

Comment:
> does it expose the plugin's internals too much?

The internals **need** to be exposed - customers must know what parameters and values are allowed. After some "staging" period it needs to be eventually covered by API stability guarantee. A struct in api.go would be more appropriate here.

String is the easiest for me to parse for now until we find the right set of parameters to expose.